From a4500ae7f25221674c6dffb25f4116549e784dd9 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 26 Feb 2021 09:56:15 +0800
Subject: [PATCH] arm64/configs: merge bifrost configs into
 rockchip_linux_defconfig

The GPU driver and dts had uploaded some CLs to support it, as below:

driver:
a51983989207 ("MALI: midgard: change kbase_platform_driver.driver.name to "midgard"")
a9f72d821275 ("MALI: bifrost: remove 'of_device_id' instances related to midgard")
5a1b8a4b4b12 ("MALI: midgard: add and include rename.h")

dts:
a400cac16883 ("arm64: dts: rockchip: rk3568: gpu: only use "arm,mali-bifrost"...")

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Change-Id: I2dfdf09e7251e4efa8c80fb152cd189a147f917e
---
 arch/arm64/configs/rockchip_linux_bifrost.config | 12 ------------
 arch/arm64/configs/rockchip_linux_defconfig      |  3 +++
 2 files changed, 3 insertions(+), 12 deletions(-)
 delete mode 100644 arch/arm64/configs/rockchip_linux_bifrost.config

diff --git a/arch/arm64/configs/rockchip_linux_bifrost.config b/arch/arm64/configs/rockchip_linux_bifrost.config
deleted file mode 100644
index 1eac5c14ff91..000000000000
--- a/arch/arm64/configs/rockchip_linux_bifrost.config
+++ /dev/null
@@ -1,12 +0,0 @@
-CONFIG_MALI_BIFROST=y
-# CONFIG_MALI_MIDGARD is not set
-# CONFIG_MALI_ARBITER_SUPPORT is not set
-CONFIG_MALI_BIFROST_DEVFREQ=y
-# CONFIG_MALI_BIFROST_DMA_FENCE is not set
-# CONFIG_MALI_BIFROST_ENABLE_TRACE is not set
-# CONFIG_MALI_BIFROST_EXPERT is not set
-# CONFIG_MALI_BIFROST_GATOR_SUPPORT is not set
-# CONFIG_MALI_DMA_BUF_LEGACY_COMPAT is not set
-# CONFIG_MALI_DMA_BUF_MAP_ON_DEMAND is not set
-# CONFIG_MALI_GEM5_BUILD is not set
-CONFIG_MALI_PLATFORM_NAME="rk"
diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 139c74ba9bad..c051373cf549 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -333,6 +333,9 @@ CONFIG_MALI_PLATFORM_THIRDPARTY=y
 CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI_DEBUG=y
 CONFIG_MALI_PWRSOFT_765=y
+CONFIG_MALI_BIFROST=y
+CONFIG_MALI_BIFROST_DEVFREQ=y
+CONFIG_MALI_PLATFORM_NAME="rk"
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-- 
2.35.3

